### ISSUE TITLE
BUG - Sev-3 - All - Alert component has duplicate heading accessibility roles when not expandable

### ISSUE THREAD

**Issue by oddballdave** [department-of-veterans-affairs/va-mobile-library#620]
## What happened?
Found while running unit tests in the VA Mobile App. Several unit tests fail with the Alert component when calling `screen.getByRole('heading')` because both the Alert and child Text components have `role="heading"` properties.

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Steps to Reproduce

Run the unit tests in the VA Mobile App by executing `yarn test`.

## Desired behavior

The VA Mobile App unit tests should not fail. The Alert component should not specify the `role="heading"` property. Instead, this should be handled by the child Text component by using the `variant="heading"` property.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: Low 
- Frequency: Low 

- 3 - Low

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [x] Steps to reproduce are defined
- [x] Desired behavior is added
- [x] Labels added (front-end, back-end, relevant feature, accessibility, etc)
- [x] Estimate of 1 added (for front-end tickets)

**Comment by jennb33** [comment]
1/23/2025 - the deployment scheduled for 1/22/2025 was cancelled. Rolling this ticket to Sprint 10, for the next deployment (scheduled for 2/5/2025, go-live on 2/11/2025 in the App Stores)
